### PR TITLE
chore: Incremental Docs Reorg Pt. 1

### PR DESCRIPTION
**Pull request by tssweeney**
This PR:
* Re-organizes the sidebar for the product overview to get closer to what we want in the long run
* Adds some emojies
* Puts integrations top level
* Reworks some of the index pages to be more professional

This PR does not:
* Break any existing links
* Remove any content

### CODE REVIEW

File: docs/docs/guides/integrations/index.md [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
 
+:::success[Automatic Tracking]
+In most cases, all you need to do is call `weave.init()` at the top of your script or program in order for Weave to automatically patch and track any of these libraries!
+:::
+
+Weave provides automatic logging integrations for popular LLM providers and orchestration frameworks. These integrations allow you to seamlessly trace calls made through various libraries, enhancing your ability to monitor and analyze your AI applications.
+
+## LLM Providers
+
+LLM providers are the vendors that offer access to large language models for generating predictions. Weave integrates with these providers to log and trace the interactions with their APIs:
```
Comment (andrewtruong) [inline docs/docs/guides/integrations/index.md:11]: Do you need the first sentence?  Sounds like GPT fluff

File: docs/docs/guides/integrations/index.md [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
+- **[Groq](/guides/integrations/groq)**
+- **[Open Router](/guides/integrations/openrouter)**
+- **[LiteLLM](/guides/integrations/litellm)**
+
+
+**[Local Models](/guides/integrations/local_models)**: For when you're running models on your own infrastructure.
+
+## Frameworks
+
+Frameworks help orchestrate the actual execution pipelines in AI applications. They provide tools and abstractions for building complex workflows. Weave integrates with these frameworks to trace the entire pipeline:
```
Comment (andrewtruong) [inline docs/docs/guides/integrations/index.md:29]: Same here.  Not sure on the value of the first 2 sentences

File: docs/docs/guides/tools/index.md [hunk 3]
```diff
@@ -1,4 +1,21 @@
-# Tools
+# Tools & Utilities
 
-- [serve](/guides/tools/serve): Serve Weave ops and models
-- [deploy](/guides/tools/deploy): Deploy Weave ops and models to various targets
+Weave is developing a set of tools and utilities to help with your workflow and deployment process for AI applications. These are currently in early alpha stages and subject to change. Here's an overview of what we're working on:
+
+## Serve
+
+[Serve](/guides/tools/serve) is an experimental feature that aims to expose your Weave ops and models as API endpoints. We're exploring possibilities such as:
```
Comment (andrewtruong) [inline docs/docs/guides/tools/index.md:7]: aims to expose -> exposes

File: docs/docs/guides/tools/index.md [hunk 4]
```diff
@@ -1,4 +1,21 @@
-# Tools
+# Tools & Utilities
 
-- [serve](/guides/tools/serve): Serve Weave ops and models
-- [deploy](/guides/tools/deploy): Deploy Weave ops and models to various targets
+Weave is developing a set of tools and utilities to help with your workflow and deployment process for AI applications. These are currently in early alpha stages and subject to change. Here's an overview of what we're working on:
+
+## Serve
```
Comment (andrewtruong) [inline docs/docs/guides/tools/index.md:5]:
```suggestion
## Serve (experimental)
```

File: docs/docs/guides/tools/index.md [hunk 5]
```diff
@@ -1,4 +1,21 @@
...
+
+[Serve](/guides/tools/serve) is an experimental feature that aims to expose your Weave ops and models as API endpoints. We're exploring possibilities such as:
+
+- Creating web services for your Weave components
+- Integrating Weave components into existing applications
+- Providing a way to test models in a more production-like setting
+
+## Deploy
```
Comment (andrewtruong) [inline docs/docs/guides/tools/index.md:13]:
```suggestion
## Deploy (experimental)
```

File: docs/docs/guides/tools/index.md [hunk 6]
```diff
@@ -1,4 +1,21 @@
...
+
+[Deploy](/guides/tools/deploy) is another alpha-stage utility we're developing to help with deploying Weave ops and models. Some potential features we're considering include:
```
Comment (andrewtruong) [inline docs/docs/guides/tools/index.md:15]: [Deploy](/guides/tools/deploy) helps with deploying Weave ops and models. Some potential features we're considering include:

File: docs/docs/guides/tracking/index.md
Comment (andrewtruong) [inline docs/docs/guides/tracking/index.md:1]: Why is the filename tracking, if it's about tracing?

**Review by andrewtruong — APPROVED** [review]
just small phrasing nits

File: docs/docs/guides/tracking/index.md
Comment (tssweeney) [reply docs/docs/guides/tracking/index.md:1]: So we don't break links. Link revisions to come in future PR. I want to get our structure reworked first, then do a 1 time name migration for the files and put in correct redirects

**Pull request merged**